include/winbond-otp.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (C) 2015 Imagination Technologies
+ * Author: Avinash Tahakik <avinash.tahakik@imgtec.com>
+ *
+ * SPDX-License-Identifier:	GPL-2.0+
+ */
+
+#ifndef WINBOND_OTP_H
+#define WINBOND_OTP_H
+    #define ETH_MAC_ADDRESS_OFFSET          0x1015 /* Ethernet MAC address offset */
```

Comment:
if we have decided to add the support for, to set WI-Fi address using U-boot in that case we will have to redefine this macro, so I think its better if we keep it here ?